src/plugins/GUITestBase/src/base_dialogs/GTFileDialog.cpp

```diff
@@ -323,6 +328,18 @@ void GTFileDialogUtils::setViewMode(ViewMode v) {
 }
 #undef GT_METHOD_NAME
 
+#define GT_METHOD_NAME "applyFilter"
+void GTFileDialogUtils::applyFilter() {
+    if (filter.isEmpty()) {
+        return;
+    }
+
+    GTGlobals::sleep(300);
```

Comment:
Do we really need this sleep? findComboBox will wait up to 30 seconds for the widget to appear

Reply:
I have added this sleep because the set of the similar functions (like `setViewMode` or `selectFile`) have sleep-wrappers like this. I do not know either this sleep is required or not. I can remove it if you want.